### ISSUE TITLE
ICO listing

### ISSUE THREAD

**Issue by AlexeyAnshakov** [webRunes/WRIO-InternetOS#1274]
Here are some ICO platform links
https://icodrops.com/
https://www.icocountdown.com/
https://icobench.com/publish
https://marketmakers.io/ico-solutions/
https://tokenmarket.net/
https://www.kickico.com/
https://komodoplatform.com/en
http://ratei.co/
https://www.tokendata.io/upcoming
https://www.trackico.io/
https://icoholder.com
https://icowatchlist.com/
https://icomarks.com
https://icomonitor.pro
https://www.coingraph.io
https://bubniv.ru/ico/ take urls from their website
We have to choose one of them (@AlWaleed84  please find more and contact them) or create our own @michbil : how much time it would take?

**Comment by AlexeyAnshakov** [comment]
Perhaps it's possible to be listed on all of them.
More links
https://duckduckgo.com/?q=ico+platform&ia=web

**Comment by AlexeyAnshakov** [comment]
https://wizard.oracles.org/

**Comment by AlexeyAnshakov** [comment]
@michbil what do you think about https://wizard.oracles.org ?

**Comment by AlexeyAnshakov** [comment]
https://komodoplatform.com/en

**Comment by ghost** [comment]
has one been chosen?

**Comment by AlexeyAnshakov** [comment]
@AlWaleed84 did you get an answer from any of them?

**Comment by AlexeyAnshakov** [comment]
http://ratei.co/

**Comment by ghost** [comment]
icobox i hear is a good one
ᐧ
j.

**Comment by AlexeyAnshakov** [comment]
But very expensive.

**Comment by AlexeyAnshakov** [comment]
We'll contact https://coralfundraiser.com/#contact-us after WP and intro completion.

**Comment by AlexeyAnshakov** [comment]
@michbil can be useful https://github.com/gmofficial/gamemachine-token as an example.

**Comment by AlexeyAnshakov** [comment]
Request template https://docs.google.com/document/d/1XdMjQvnXyjE12JS8coH-eMcHFouFT9Gksa_2Vd_BKuw/edit?usp=sharing

**Comment by ghost** [comment]
ok
ᐧ
j.

**Comment by AlexeyAnshakov** [comment]
@AlWaleed84 please contact them and learn more about their services and terms https://crypto4all.com/initial-coin-offering-token-crowdsale/. Thanks

**Comment by AlexeyAnshakov** [comment]
https://www.tokendata.io/upcoming

**Comment by AlexeyAnshakov** [comment]
Hoticolist.com

**Comment by ghost** [comment]
Hi There

Do we have finalized dates etc - Has that summary sheet been
updated/finalized?
Once that is done please post it here so that all information is consistent
throughout the listings

I think it's important to start to post on these ICO listing sites ASAP

Thanks
ᐧ

j.

**Comment by AlexeyAnshakov** [comment]
Not yet. We have to find an ICO manager first.

**Comment by AlexeyAnshakov** [comment]
@AlWaleed84 https://tokenget.com  please contact them for details.

**Comment by AlexeyAnshakov** [comment]
https://github.com/Scanate/UltimateICOCalendar

**Comment by AlexeyAnshakov** [comment]
https://hackernoon.com/top-10-ico-listing-sites-ico-alert-679191242da0

**Comment by AlexeyAnshakov** [comment]
Roy
```
Wiserico listing site charges 2-3 eth to get listed , I'm not sure they are worth it but anyone thinks differently? https://wiserico.com/
```

**Comment by RoySasson** [comment]
https://wiserico.com/ -   charges 2-3 eth 
https://tokentops.com/ -   charges  1-5 eth
https://www.trackico.io/   - in progress \ review

**Comment by AlWaleed84** [comment]
Lets have a look at https://www.smithandcrown.com/

They also do research on each project. 


AlWaleed K. AlShehail

**Comment by ghost** [comment]
Hey there 
Mi shared a list of 20+ sites with cost and contact info 
Do you have that link still? 

https://docs.google.com/spreadsheets/d/1RR6NQEaVs74ahHj9lc5PpzuDkqnLxOzJRkIm4voW088/edit?usp=sharing
ᐧ

j.

**Comment by jaezed** [comment]
https://docs.google.com/spreadsheets/d/1RR6NQEaVs74ahHj9lc5PpzuDkqnLxOzJRkIm4voW088/edit?usp=sharing
ᐧ

j.

**Comment by AlexeyAnshakov** [comment]
Ok, thanks, no need to keep in the doc. Moved here
Coinschedule.com
CoinGecko.com
Cyber.Fund
ICOCrowd.com
ICOCountdown.com
ICO-List.com
Happycoin.club
Tokenmarket.net
TokenSaleCalendar.com
http://icodaily.ru/ico-calendar/
https://icowatchlist.com/
https://www.icoalert.com/
https://www.smithandcrown.com/icos/
https://icotracker.net/
https://www.trackico.io/
http://icopitch.com/
http://whattomine.com/
http://icoage.com/
https://www.investitin.com/ico/
http://ratei.co
http://icodude.com/
http://icocrowd.com/calendar/
icochecker.com/
https://icodaily.net/
https://topicolist.com/
https://tokenmarket.net/ico-calendar
https://icobazaar.com/list
https://www.icoship.co/
https://icoscroll.com/
m.cryptodetails.com/ico/
https://www.applancer.co/ico-list
https://www.cryptodetails.com/ico/
http://www.cryptomorrow.com/
https://icotimeline.com/services/
https://tokentops.com/
https://topicolist.com/
https://coinformant.com/cryptocurrency-icos/list-ico-asset/
blockstarter.co/
ico.newsbtc.com/
http://ico-cryptocurrency.com/
https://bl4nkcode.info/ico_list
https://www.coinhills.com/ico/
https://timesofethereum.com/
https://icomagazine.com/submit-post/
https://cryptostreet.co/submit-ico/
https://icodrops.com/submit-ico/
https://cryptocanucks.com/ico-submission/
https://www.icodragon.com/submit-ico/
icocrowd.com/send-a-press-release/
kicktrend.com/submit-ico-info/
https://www.icoranker.com/submit-your-ico/
initialcoinofferingslist.com/submit-ico/
icowatchers.com/submit-listing/
https://icopressrelease.com/submit-your-ico-press-release-for-free/
https://www.icovend.com/en/login
https://www.coinist.io/submission/
https://icosignal.com/
www.cryptocoinindonesia.com/submit-ico
altcoinalerts.com/
https://cryptorated.com/
https://ICOToday.io.
https://icobench.com/publish
https://marketmakers.io/ico-solutions/

**Comment by AlexeyAnshakov** [comment]
https://foundico.com/

**Comment by AlexeyAnshakov** [comment]
Icorating.

Icoalert.

IcoTracker.

CoinsCat.

Tokendata.

Hubcoin.

FoundIco.

Coinhills.

Icobench.

ICOAdvert.

Icolighthouse.

Icolert.

Foxico.

ICO Pro.

Ico holder.

ICORanker.

Trackico.

**Comment by RoySasson** [comment]
Can I publish the date we settled July 31th? 

----- הודעה מקורית -----
מאת: "Alexey Anshakov" <notifications@github.com>
נשלח: ‎11/‎05/‎2018 23:10
אל: "webRunes/WRIO-InternetOS" <WRIO-InternetOS@noreply.github.com>
עותק: "Roy Sasson" <roeysa@gmail.com>; "Assign" <assign@noreply.github.com>
נושא: Re: [webRunes/WRIO-InternetOS] ICO listing (#1274)

Icorating.
Icoalert.
IcoTracker.
CoinsCat.
Tokendata.
Hubcoin.
FoundIco.
Coinhills.
Icobench.
ICOAdvert.
Icolighthouse.
Icolert.
Foxico.
ICO Pro.
Ico holder.
ICORanker.
Trackico.

**Comment by AlexeyAnshakov** [comment]
No. Like i said: the date is only as an example

**Comment by RoySasson** [comment]
Some sites doesnt let you write TBA , so must choose any date 

----- הודעה מקורית -----
מאת: "Alexey Anshakov" <notifications@github.com>
נשלח: ‎11/‎05/‎2018 23:36
אל: "webRunes/WRIO-InternetOS" <WRIO-InternetOS@noreply.github.com>
עותק: "Roy Sasson" <roeysa@gmail.com>; "Assign" <assign@noreply.github.com>
נושא: Re: [webRunes/WRIO-InternetOS] ICO listing (#1274)

No. Like i said: the date is only as an example

**Comment by AlexeyAnshakov** [comment]
If we can update it later - ok, set 31 July.

**Comment by RoySasson** [comment]
https://www.trackico.io/ *registered *
https://foundico.com *- registered *
wiserico - Asks for 2/3 eth
coingeck.com - *registered but not online yet* ( emailed them for that
issue )
https://topicolist.com - Asks for 500euro
https://cryptorated.com - *registered   *might be accepting tokens for
highlight promotion - email discussion
icomarks - *already registered*
icobench - *already registered*

2018-05-12 9:47 GMT+02:00 Alexey Anshakov <notifications@github.com>:

**Comment by AlexeyAnshakov** [comment]
Please improve profiles like https://www.trackico.io/ico/wrio-internet-os/#tab-rating

**Comment by RoySasson** [comment]
Didn't fuly understand - improve profiles to be like as in trackico , or
trackico needs to be improved ?

2018-05-12 19:53 GMT+02:00 Alexey Anshakov <notifications@github.com>:

**Comment by RoySasson** [comment]
Sorry Didn't fully understand - improve profiles to be like as in trackico , or
trackico needs to be improved ?

**Comment by AlexeyAnshakov** [comment]
We need to improve our rating on all those services.

**Comment by AlexeyAnshakov** [comment]
https://foundico.com/ico/wrio-internet-os.html Soft cap isn't currect. We have KYC
Location: Czech Republic

**Comment by RoySasson** [comment]
Should be 700k usd soft cap ? Equals to 20 mln wgd

**Comment by AlexeyAnshakov** [comment]
About 600k

**Comment by RoySasson** [comment]
Updated that info :
https://foundico.com/ico/wrio-internet-os.html Soft cap isn't currect. We have KYC
Location: Czech Republic

If we want to get a better score on their site ,2 team members needs to do KYC .

**Comment by AlexeyAnshakov** [comment]
they added us? Or you? We won't do any KYC, so it's better to delete us from this site than have a low score. Anyway they are little known.

**Comment by RoySasson** [comment]
I added us , our score is ok but won't be the highest

**Comment by AlexeyAnshakov** [comment]
5 isn't ok. 
>Tokens for sale: | 387.500.000 WGD

310 mln.

**Comment by AlexeyAnshakov** [comment]
They added us http://betaico.com/wr must be improved

**Comment by RoySasson** [comment]
https://www.smithandcrown.com/sale/  *registered*
http://altcoinalerts.com/ asks for 300$ to register 
coingecko.com *registered* 
icotoday.io - listed 2 times but failed to submit , emailed their support

**Comment by AlexeyAnshakov** [comment]
Can't find at https://www.smithandcrown.com/sale/

**Comment by RoySasson** [comment]
strange, can't see it too - checking 
https://icorating.com - contacted them for getting listed

**Comment by RoySasson** [comment]
https://icocrunch.io - registered , not listed yet

**Comment by AlexeyAnshakov** [comment]
Please check all links above, register everywhere and create a list with our rating.

**Comment by AlexeyAnshakov** [comment]
https://docs.google.com/spreadsheets/d/1MYcOGR6_bB6GqIf_yCeSkHQrGHhCuNJ6-Eo03imKjU4/edit?ts=5b1916bf#gid=0
Please add all links above.

**Comment by AlexeyAnshakov** [comment]
Coinschedule.com
CoinGecko.com
Cyber Fund
ICOCrowd.com
ICOCountdown.com
ICO-List.com
Tokenmarket.net
TokenSaleCalendar.com
https://icostock24.com/content26/

**Comment by RoySasson** [comment]
Will add these sites .
